src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -30,20 +30,20 @@
     /**
      * Initializes a ModelManager with the given addressBook and userPrefs.
      */
-    public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs) {
+    public ModelManager(ReadOnlyAddressBook addressBook, Attendance attendance, ReadOnlyUserPrefs userPrefs) {
```

Comment:
Changing this constructor will require updating `FilterCommandTest` and `SortCommandTest`as well.

Reply:
Done! I've pulled the from the remote branch and updated the failing tests. Turns out TrainingTest and SelectCommandTest were broken too!